test/dist_autograd_test.py

```diff
@@ -125,7 +132,7 @@ def world_size(self):
     def init_method(self):
         return INIT_METHOD_TEMPLATE.format(file_name=self.file_name)
 
-    @dist_init
+    @dist_init(setup_model_parallel=True)
```

Comment:
https://github.com/pytorch/pytorch/pull/27940 will land soon, we shouldn't need to specify the default arg after that.

Reply:
Yes, I will rebase after that lands.